.github/workflows/ci.yaml

```diff
@@ -51,7 +51,9 @@ jobs:
           docker images
       - name: Move cache
         run: |
-          rm -r /tmp/.buildx-cache
```

Comment:
I think you can just do `rm -rf /tmp/.buildx-cache`, it should ignore if it's not a directory

Reply:
yes, that should also work - just wanted to keep the check clear

Reply:
I vote for -rf.  This is the Linux admin way.  A lot quicker to read then the if statement.

Reply:
I'll switch to -rf then